Json.cpp

```diff
@@ -91,19 +91,33 @@ std::map<std::string, std::string> Json::ParseString(const std::string &input)
 
     size_t pos = 0;
     std::string token;
+
     while ((pos = text.find("\"")) != std::string::npos)
     {
         token = text.substr(0, pos);
         token = Trim(token);
+        if (isdigit(token[1]))
+        {
+            if (std::stof(token) <= 0)
+            {
+                throw std::runtime_error("Invalid value");
+            }
+        }
+
         data.push_back(token);
         text.erase(0, pos + 1);
     }
     for (size_t i = 0; i < data.size() / 2; i++)
     {
         mapedData[data[2 * i]] = data[2 * i + 1];
     }
+    if (mapedData.size() != 4)
```

Comment:
Annak `mappedData`-nak kéne lennie.

Reply:
Ha a map nem négy elemet tartalmaz, akkor miért invalid a Json? 
Tudnia kell beolvasni kisebb és nagyobb JSON fájlokat is.